Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/PhysicsPicking.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Editor/Picking.h>
+
+namespace EMotionFX
+{
+    static constexpr float PickingMargin = 0.01f;
+
+    class PhysicsPicking : public Picking
```

Comment:
PhysicsPicking feels like the wrong name to me as this is about picking the skeleton and colliders (where 3 out of 4 are indeed used by something physics related). Can we remove that base class and just call it EMFX::Picking?

Reply:
The idea was that it would be the picking that would be active when you're in the physics application mode in the animation editor, and possibly you would want other picking modes for other application modes. That was the point of the base class, so you could add other picking modes later and have some logic for deciding which picking mode was active. Might be just overthinking it though.